src/main/java/uk/gov/hmcts/reform/em/npa/service/dto/external/redaction/RedactionDTO.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.em.npa.service.dto.external.redaction;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import javax.validation.constraints.NotBlank;
+import java.util.UUID;
+
+@Getter
+@EqualsAndHashCode
+@ToString
+public class RedactionDTO {
+    // Json ignore for which fields?
+    @NotBlank(message = "UUID cannot be Blank")
+    private UUID id;
+
+    @NotBlank(message = "pageNumber cannot be Blank")
+    private int pageNumber;
```

Comment:
As discussed, this will be null for image files in media viewer as they are single pages

Reply:
Agreed that front end will set this to one in case of images